README.md

```diff
@@ -1,127 +1,220 @@
+2. Web3.js or Ethers.js
+
+###  📌 Use Cases
 
-**IContract interface**
+This help content focuses on the following use cases. 
 
-* Task<string> CallMethod(string methodName, object[] arguments, string gas = null)*<br>
-Use for call methods of a contract deployed to blockchain.
-Returns string with transaction hash.
+1. Connecting to a Web3 Wallet (MetaMask) and Authenticating a user
+
+2. Performing Updates to NFTs by interacting with the blockchain and calling smart functions.
+
+
+<!-- 
+1. Examples Scripts, 
+*  The *Scripts* folder contains a wide range of useful scripts for initializing sessions, creating and managing Game NFTs and Scenes and authenticating sessions via a Web3 Wallet e.g. Metamask. 
+`ContractMessage` scripts for retrieving information for and managing NFTs  e.g. Game Characters on the blockchain.  ERC-721 and ERC-1155 
+`DTO` scripts for handling TransferObject events and accessing logs.
+`ERC-20`folder contains example script for minting new NFTs e.g. characters
+`ERC-721` folder contains example scripts for managing ownership of NFTs.
+`WearableNFTExample` folder contains example scripts for minting and managing NFT character apparel e.g. hats. 
+`ConnectionController` and `CustomSceneManager` ensure ongoing authentication via MetaMask and displaying game scenes. 
+to allow smart contract interfacing with the for building NFTs to the Blockchain.  scripts to ERC721 and ERC1155 scriptsScripts that can be used  -->
+
+## 👝 01 Connect Web3 Wallet
+
+Connecting to an Ethereum i.e. MetaMask wallet provides a link between a wallet address and a user's game account.
+
+1. Create an instance of a `Web3` class and call `Initialize` method after successful login in metamask
 
 ```c#
-string receipt = await contract.CallMethod("methodName", new object[0]);
+string provider_url = "<ethereum node url>";
+		
+Web3 web3 = new Web3(provider_url);
+web3.Initialize();
 ```
----
-* Task<Transaction> GetTransactionInfo(string receipt)*<br>
-Use for get transaction details like status, nonce and etc.
-Returns transaction, which is a Nethereum DataContract containing all the needed information about transaction. 
+
+2. Login via MetaMask is required to authenticate the user.
+
+- Call the `SignMethod` to trigger Signing via MetaMask. 
 
 ```c#
-Transaction trx = await contract.GetTransactionInfo(receipt);
+string message = "Hi I am a message !"
+string signature = await web3.Sign(message);
 ```
----
-* Task<TReturnType> GetData<FieldData, ReturnType>(FieldData requestData = null)*<br>
-To get data from mappings and call contract methods that no need of mining use this method.
 
-You need to prepare DTO class based on the arguments of a contract method.
+- Continue by deploying the [Unreal Engine (backend)](https://github.com/Ankr-network/game-unreal-sdk#-01-connect-wallet) 
+
+- The backend returns a JSON object (payload) with the `session_id` and `URI`.
+
+- `Web3.Sign(string)` returns the signature.
```

Comment:
Signing is happening via 
AnkrSignatureHelper.Sign: 
`Task<string> Sign(string messageToSign)`
 and AnkrSignatureHelper.CheckSignature:
`string CheckSignature(string messageToCheck, string signature)`